### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**
- [DEMO LINK](https://yelnikov-andrii.github.io/layout_catalog/)
- [TEST REPORT LINK](https://yelnikov-andrii.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by EnjSadman — CHANGES_REQUESTED** [review]
Не працюють демо-лінки. Спробуй ще раз зробити деплой

**Comment by yelnikov-andrii** [comment]
> Не працюють демо-лінки. Спробуй ще раз зробити деплой

fixed

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,370 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img class="header__logo" src="./images/logo.png" alt="logo">
```
Comment (Anna-Tkachenko) [inline src/index.html:11]: It should be the link

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,370 @@
...
+          </li>
+          <li class="nav__link">
+            <a class="nav__item" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__item" href="#">
+              Смартфоны и телефоны  
+            </a>
```
Comment (Anna-Tkachenko) [inline src/index.html:27]: Set correct hrefs for the links

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,370 @@
...
+    <main class="container">
+
+      <div class="card" data-qa="card">
+        <img class="card__img" src="./images/imac.jpeg" alt="image of device">
+        <h3 class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <span class="card__code">
+            Код товара: 195434
+        </span>
```
Comment (Anna-Tkachenko) [inline src/index.html:69]: Fix indents
```suggestion
        <span class="card__code">
          Код товара: 195434
        </span>
```

File: src/styles/blocks/header.scss [hunk 4]
```diff
@@ -0,0 +1,79 @@
+.header {
+  display: flex;
+  justify-content: space-between;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/header.scss:3]: Will be better to use variable for correct indents between the cards (it should fix the tests)

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      justify-content: space-between;
+      list-style: none;
+      width: 100%;
+      margin: 0;
+      padding: 0;
+    }
+
+    &__item {
+      font-weight: 500;
+      font-size: 12px;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/header.scss:26]: Move it to nav.scss file

File: src/styles/blocks/main.scss [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/main.scss:16]: Move this styles to container.css and card.scss files

File: src/styles/blocks/main.scss [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  
+  background-image: url(../../images/star-active.svg);
+}
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/main.scss:93]: Use cycle instead of 5 selectors
Move this styles to stars.scss file

File: src/styles/blocks/main.scss [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+  
+.card__price-title {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+  
+.card__price-value {
+  font-size: 16px;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/main.scss:112]: Move this styles to card.scss file

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Fix comments and try to pass tests

File: src/index.html [hunk 9]
```diff
@@ -4,9 +4,371 @@
...
+          <li class="nav__link">
+            <a class="nav__item is-active" href="/">
+              Apple
+            </a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__item" href="/">
+              Samsung
+            </a>
+          </li>
```
Comment (Anna-Tkachenko) [inline src/index.html:25]: Set correct href for all links

File: src/index.html [hunk 10]
```diff
@@ -4,9 +4,371 @@
...
+          <li class="nav__link">
+            <a class="nav__item" href="/">
+              Смартфоны и телефоны  
+            </a>
+          </li>
+          <li class="nav__link">
+            <a  class="nav__item" 
+                data-qa="nav-hover"    
+                href="/">
```
Comment (Anna-Tkachenko) [inline src/index.html:34]: Fix indents

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+  width: $cardsize;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: $gapY;
+  margin-right: $gapX;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/card.scss:13]: You can't set width and margins for BEM block

File: src/styles/blocks/main.scss [hunk 12]
```diff
@@ -0,0 +1,12 @@
+
+@import "../utils/variables.scss";
+
+.main {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  margin: 0 auto;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/main.scss:9]: You can't set width and margin for BEM block

File: src/styles/blocks/nav.scss [hunk 13]
```diff
@@ -0,0 +1,81 @@
+@import "../utils/variables.scss";
+
+.nav {
+  width: 756px;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/nav.scss:4]: Same here

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Fix comments, check font styles and indents for your page and try to pass tests

File: src/styles/blocks/card.scss [hunk 14]
```diff
@@ -0,0 +1,122 @@
+@import "../utils/variables.scss";
+
+.main__card {
+  width: $cardsize;
+  margin-bottom: $gapY;
+  margin-right: $gapX;
+}
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/card.scss:7]: Move this styles to main.scss file

File: src/styles/blocks/header.scss [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+
+  &__logo {
+    align-self: center;
+    height: 40px;
+  }
+}
+
+.header__nav {
+  width: 756px;
+}
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/header.scss:16]: Use nesting

File: src/styles/blocks/main.scss [hunk 16]
```diff
@@ -0,0 +1,15 @@
+
+@import "../utils/variables.scss";
+
+.page__main {
+  margin: 0 auto;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/main.scss:5]: Move this styles to page.scss file

File: src/styles/style.scss [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+@import url(./blocks/stars.scss);
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.page {
+  margin: 0;
+  background: #fff;
+}
```
Comment (Anna-Tkachenko) [inline src/styles/style.scss:16]: Move this styles to page.scss
You can't set margins to BEM block

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**